src/new-components/banner/banner.stories.mdx

```diff
@@ -0,0 +1,234 @@
+    )
+}
+
+export function Template({ tone, title, description, action }) {
+    return (
+        <Stack space="large" maxWidth="medium">
+            <Banner
+                tone={tone}
+                icon={Icon()}
+                title={title}
+                description={description}
+                action={action}
+            />
+        </Stack>
+    )
+}
+
+export function BannerExamples() {
+    return (
+        <Stack space="medium" align="start">
+            <Stack space="large">
+                <BannerWrapper
+                    tone="promotion"
+                    icon={StarIcon()}
+                    title="Need more projects?"
+                    description="This workspace has used 5 of its 5 project limit. Upgrade to Pro for more."
+                    action={getButton('Upgrade')}
+                />
+                <BannerWrapper
+                    tone="info"
+                    icon={ArchiveIcon()}
+                    title="This project is archived."
+                    description="Members can view but not edit."
+                    action={getButton('Unarchive project')}
+                />
+                <BannerWrapper
+                    tone="info"
+                    icon={ArchiveIcon()}
+                    title="This project is archived."
+                    description="Members can view but not edit."
+                />
+                <BannerWrapper
+                    tone="info"
+                    icon={EyeIcon()}
+                    title="Youʼre previewing this project."
+                    action={getButton('Join project')}
+                />
+            </Stack>
+        </Stack>
+    )
+}
+
+# Banner
+
+A simple banner component meant to be used to _inform_ the user of promotional content or disclaimers.
+
+If you're intending to _alert_ the user of a certain error or warning condition, consider using the `Alert` component instead.
+
+<Canvas>
+    <Story
+        name="Playground"
+        parameters={{ docs: { source: { type: 'code' } } }}
+        argTypes={{
+            tone: {
+                options: ['info', 'promotion'],
+                control: { type: 'inline-radio' },
+                defaultValue: 'info',
+            },
+            icon: { control: false },
+            title: {
+                control: { type: 'text' },
+                defaultValue: 'Title of the Banner',
+            },
+            description: {
+                control: { type: 'text' },
+                defaultValue: 'Description of the Banner.',
+            },
+            action: { control: false },
+            id: { control: false },
+        }}
+    >
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+## Examples
+
+<Canvas>
+    <Story
+        name="Main demo"
+        parameters={{
+            docs: { source: { type: 'code' } },
+            chromatic: { disableSnapshot: false },
```

Comment:
Nice! I like that you realized that we want to enable visual regression testing in this kind of story that showcases the different look and feels all at once. It's good to know that that was somehow clear, even though I have not documented yet our "Chromatic best practices".

Reply:
I mean, thank you for putting a lot of respect and trust in me, but I must admit that I would've put this even if the story didn't have all the "variants" 😅

Actually, now that I think about it, maybe this `chromatic: { disableSnapshot: false },` should go even on the dark mode story that I added at the end of this file. Why. not?